tip-wlan/charts/kafka/resources/config/certs/keystore_creds

```diff
@@ -1 +1 @@
-DUMMY_PASSWORD
\ No newline at end of file
+mypassword
```

Comment:
Please revert this to DUMMY_PASSWORD, (until we change this file into a ConfigMap/Secret and read it from override file)